build/lib/compilation.ts

```diff
@@ -104,11 +104,14 @@ export function transpileTask(src: string, out: string, swc: boolean): () => Nod
 			.pipe(transpile())
 			.pipe(gulp.dest(out));
 	};
+
+	task.taskName = `transpile-${path.basename(src)}`;
+	return task;
 }
 
 export function compileTask(src: string, out: string, build: boolean, options: { disableMangle?: boolean } = {}): () => NodeJS.ReadWriteStream {
```

Comment:
Same for this. It looks like the return type should be 
```
{
    (): NodeJS.ReadWriteStream;
    taskName: string;
}
```
 for both of these methods.

Reply:
> Same for this. It looks like the return type should be
> 
> ```
> {
>     (): NodeJS.ReadWriteStream;
>     taskName: string;
> }
> ```
> 
> for both of these methods.

Using this existing type `StreamTask` from  https://github.com/microsoft/vscode/blob/85daa35afb86202badbee79532537bf226961f25/build/lib/task.ts#L17 seems more appropriate?